fdbserver/Ratekeeper.actor.cpp

```diff
@@ -1066,7 +1070,7 @@ void updateRate(RatekeeperData* self, RatekeeperLimits* limits) {
 			}
 		}
 
-		int64_t storageQueue = ss.lastReply.bytesInput - ss.smoothDurableBytes.smoothTotal();
+		int64_t storageQueue = ss.lastReply.bytesInput + ss.smoothPendCompactionBytes.smoothTotal(); - ss.smoothDurableBytes.smoothTotal();
```

Comment:
I don't think we should change the definition of storageQueue. 

We should add a `rocksPendingCompactionQueue` by using RocksDBMetrics's value `EstPendCompactBytes` , monitor it as RK monitors tLog's queue, and calculate tpsLimit based on `rocksPendingCompactionQueue `. If the tpsLimit is the smallest, RK will throttle with a new reason rocks_pending_compaction_queue (LimitID=12).